roles/lib_openshift/src/doc/process

```diff
@@ -0,0 +1,80 @@
+# flake8: noqa
+# pylint: skip-file
+
+DOCUMENTATION = '''
+---
+module: oc_process
+short_description: Module to process openshift templates
+description:
+  - Process openshift templates programmatically.
+options:
+  state:
+    description:
+    - Currently present is only supported state.
+    default: present
+    choices: ["present", "absent", "list"]
```

Comment:
Docs: Is this accurate? It looks like `present` and `list` states are supported.